app/cells/placeholder_users/row_cell.rb

```diff
@@ -46,11 +46,20 @@ def button_links
     end
 
     def delete_link
-      return nil unless PlaceholderUsers::DeleteContract.deletion_allowed?(User.current)
+      if PlaceholderUsers::DeleteContract.deletion_allowed?(placeholder_user,
+                                                            User.current,
+                                                            table.user_allowed_service)
 
-      link_to '',
-              deletion_info_placeholder_user_path(placeholder_user),
-              class: 'icon icon-delete'
+        link_to deletion_info_placeholder_user_path(placeholder_user) do
+          "<span class=\"tooltip--left\" data-tooltip=\"#{I18n.t('placeholder_users.delete_tooltip')}\"><i class=\"icon icon-delete\"></i></span>".html_safe
+        end
+      else
+        "<span class=\"tooltip--left\" data-tooltip=\"#{I18n.t('placeholder_users.right_to_manage_members_missing')}\"><i class=\"icon icon-help2\"></i></span>".html_safe
```

Comment:
```suggestion
        content_tag :span, op_icon('icon icon-help2'), class: 'tooltip--left', data: { tooltip: I18n.t('placeholder_users.right_to_manage_members_missing') }
```

Avoid interpolation hell and use a content_tag instead :grimacing:

Reply:
Thanks, I'll do. I see your point.

Reply:
@ulferts 

> Taking a look at the PR, I couldn't come up with a reason why to introducing this check for placeholder users but not for users. And if there is no compelling reason to do it for users, why do it for placeholder users. Can you shed some light on this for me, @wielinde ?

There is no global permission for deleting users as there is for managing placeholder users. Only admins can delete users. And admins always have full access to all projects and can manage the members of all projects. So I don't see a point of adding that logic to users. And I don't see that we will soften our policy in regards to deleting users in the future (we will not give normal users that right).

Reply:
I see your point, @wielinde